src/SFML/Audio/SoundStream.cpp

```diff
@@ -498,4 +483,31 @@ void SoundStream::clearQueue()
         alCheck(alSourceUnqueueBuffers(m_source, 1, &buffer));
 }
 
+
+////////////////////////////////////////////////////////////
+void SoundStream::launchStreamingThread(Status threadStartState)
+{
+    m_isStreaming = true;
+    m_threadStartState = threadStartState;
+
+    assert(!m_thread.joinable());
+    m_thread = std::thread(&SoundStream::streamData, this);
+}
+
+
+////////////////////////////////////////////////////////////
+void SoundStream::awaitStreamingThread()
+{
+    // Request the thread to join
+    {
+        std::scoped_lock lock(m_threadMutex);
+        m_isStreaming = false;
+    }
+
+    if (m_thread.joinable())
+    {
+        m_thread.join();
+    }
```

Comment:
No curly braces for single statement body.

Reply:
I don't like this style decision, and I don't think it should be enforced, but I will apply the change as I don't feel very strongly about it and I'd like to see this PR merged :)

Reply:
This style can have a purpose. Removing these braces signals to the reader that this conditional shall only have a single statement inside of it which discourages others from putting more code in here which in this case is probably not a valid thing to do. It expresses more intent. It's a nice side effect that it reduces line count which lets more code fit on a screen which can aid in reading the code.

I do think this style depends on the existence of an auto formatter so that code like this:

```cpp
if (condition)
    do_thing();
    do_another_thing();
```

gets turn into this

```cpp
if (condition)
    do_thing();
do_another_thing();
```

so that no one is confused about the scope of that conditional. In the absence of an auto formatter (or strict code review to catch these formatting issues) it's harder to justify removing these braces.

Reply:
Counter-points:
- Consistently using braces for every language construct... 
    - prevents mistakes caused by misleading indentation
    - builds muscle memory for "good practice"
    - makes maintenance of code easier (requires less keystrokes to move some code in or out of a `if` body, for example)
    - does not require external tools like autoformatters or linters to verify correctness

But people will never agree about style. :)

Reply:
Consistency is king, thus as this is the current style, we should adhere to it.
If we want to change the style, let's have a discussion on it separately. 🙂

Reply:
I agree that adhering to the rules is good, I did perform the change as soon as requested. But I am still going to complain about rules I don't like :D 

Of course, discussing on the forum or a separate issue is much better.

Reply:
I support @vittorioromeo claim. I find this to be fragile and inconsistent:

```cpp
if (condition)
    do_thing();
else
    do_another_thing();
```
but suddenly:

```cpp
if (condition)
{
    do_thing();
}
else
{
    // a comment
    do_another_thing();
}
```

It would be better to not allow it at all or enforce this to be a one-liner single expression, non-else, if-statement.